### ISSUE TITLE
Build for iOS 9.0.2 with xcode 7.0 show "Failed to find t for Y input

### ISSUE THREAD

**Issue by MehmetBal** [cbpowell/MarqueeLabel#117]
This issue only happens with short text such as http://tinyurl.com/1c2 where the text actually fits to screen. 
This is how I used the MarqueeLabel 
```obj-c
- (void)awakeFromNib 
{
    titleLabel.marqueeType   = MLContinuous;
    titleLabel.holdScrolling    = YES;
    titleLabel.scrollDuration   = 5.0f;
    titleLabel.fadeLength       = 10.0f;
    titleLabel.leadingBuffer    = 0.0f;
    titleLabel.trailingBuffer     = 15.0f;
    [self.linkTitleLabel pauseLabel]
}
```

Any ideas on how to fix this?

**Comment by cbpowell** [comment]
I haven't had a chance to look at this yet, but just as a guess until I do - it might have something to do with doing the setup in `awakeFromNib`? If it's feasible, you could try putting that setup in your view controller's `viewDidLoad` instead.

**Comment by cbpowell** [comment]
I can't reproduce this on-device (iPhone 6, 9.0.2) or in the simulator, but I had to make some assumptions about your setup. How and where are you setting the frame of the MarqueeLabel? Directly or using AutoLayout?

Here's how I set up my test label:
<img width="1133" alt="screenshot 2015-10-04 11 30 28" src="https://cloud.githubusercontent.com/assets/593953/10269398/60904bdc-6a8b-11e5-976a-c83b7192ad38.png">

As a side note, there's no reason to immediately call `[self.linkTitleLabel pauseLabel];` if you're also setting `holdScrolling = YES`. The `pauseLabel` method is for pausing mid-animation.

**Comment by MehmetBal** [comment]
Hi,

This is how I setup with Auto layout. I am using it in a table view so I can't set it up in `<viewDidLoad>`. The *date* label is not a marquee label

<img width="974" alt="screen shot 2015-10-04 at 11 46 21 am" src="https://cloud.githubusercontent.com/assets/3094502/10269489/e2bc537e-6a8d-11e5-993a-3775b52255cf.png">
 
<img width="962" alt="2" src="https://cloud.githubusercontent.com/assets/3094502/10269639/1a914ff8-6a92-11e5-9cdf-e35b65a0cec1.png">

<img width="969" alt="3" src="https://cloud.githubusercontent.com/assets/3094502/10269642/1f4f180e-6a92-11e5-8ba7-482a1a35f1a1.png">

**Comment by cbpowell** [comment]
I believe your problem is most likely because you're using `pauseLabel` in `awakeFromNib` - see this [very similar issue on the Swift version of MarqueeLabel](https://github.com/cbpowell/MarqueeLabel-Swift/issues/15).

I suggest modifying your code in a similar fashion to [what I suggested there](https://github.com/cbpowell/MarqueeLabel-Swift/issues/15#issuecomment-162352791). Utilize the `holdScrolling` property as opposed to pausing the label.

**Comment by cbpowell** [comment]
I've pushed [release v2.4.1](https://github.com/cbpowell/MarqueeLabel/releases/tag/2.4.1) which performs the appropriate checks before allowing the `pauseLabel` logic to be performed. This should prevent the log messages you're seeing as a result of calling `pauseLabel` in `awakeFromNib`.

**However**, you'll need to modify your code to use the `holdScrolling` property as I suggested. Hope this helps!
